src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourcePane.java

```diff
@@ -150,6 +150,11 @@ public void closeTab(int index, boolean interactive, Command onClosed)
          tabPanel_.closeTab(index, onClosed);
    }
    
+   public void closeAllTabs()
+   {
+      tabPanel_.clear();
```

Comment:
Thanks -- just made the accompanying change. (I'll try to be more careful when looking at / touching code that requires synchronization between the client + server)